portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -53,7 +57,22 @@ public Duration getDuration() {
     return duration;
   }
 
-  public Integer getPriority() {
-    return priority.getPriority();
+  public TaskPriority getPriority() {
+    return priority;
+  }
+
+  // These methods provide a way to tell if two objects are both Task objects
+  // and have all the same fields
+
+  @Override
+  public boolean equals(Object other) {
+    return other instanceof Task && equals(this, (Task) other);
+  }
+
+  private static boolean equals(Task a, Task b) {
```

Comment:
This might become an issue as you explore Google Tasks integration. I think you may want to start passing task IDs from the client. I think you can have multiple tasks with the same name in Google Tasks.

Reply:
Resolved.